lute/templates/read/reading_menu.html

```diff
@@ -123,6 +123,11 @@
             Mobile
           </a>
         </li>
+        <li>
+          <a id="screen-mobile-focus" class="reading-menu-item" href="" onclick="set_screen_type('mobile-focus'); return false;">
+            Mobile Focus
```

Comment:
People will need some kind of documentation on what this means.  There is a "desktop focus", but this is different, you're changing how clicks work.  Not sure what the right place for this is, but I don't think this is it.  Hard to say.

Edit: on the other hand, this _is_ defining/refining "screen interactions".  Still feels off, and I can't explain why.

> <img width="362" alt="image" src="https://github.com/user-attachments/assets/890ca826-9ed8-4c8e-96f9-5d841fb29fa0" />